plugins/web/opentelemetry-instrumentation-user-interaction/test/userInteraction.nozone.test.ts

```diff
@@ -546,6 +562,70 @@ describe('UserInteractionInstrumentation', () => {
       });
     });
 
+    it('should not create spans from unknown events', () => {
+      fakeEventInteraction('play');
+      assert.strictEqual(
+        exportSpy.args.length,
+        0,
+        'should not export any spans'
+      );
+    });
+
+    it('should export spans for configured event types', () => {
+      registerInstrumentation({
+        eventTypes: ['play'],
+      });
+
+      fakeEventInteraction('play');
+      assert.strictEqual(exportSpy.args.length, 1, 'should export one span');
+      const span = exportSpy.args[0][0][0];
+      assertInteractionSpan(span, { name: 'play' });
+    });
+
+    it('should not export clicks if this event was not configured to be captured', () => {
```

Comment:
whenever you use word "when" or "if" ettc. you should split the declaration with one extra level
```
describe('if event is not configured', ()=> {
  it('should not export clicks', ()=> {
    ...
  });
})
```